internal/utils/marshall.go

```diff
@@ -62,3 +38,24 @@ func UnmarshallList(arg string) ([]string, error) {
 	}
 	return res, nil
 }
+
+func MarshallToString(args interface{}) (string, error) {
```

Comment:
super small nit: Marshall to Marshal everywhere?

Reply:
yes!